property.go

```diff
@@ -3,15 +3,15 @@ package opslevel
 import "fmt"
 
 type PropertyDefinitionInput struct {
-	Name   string     `json:"name" defaults:"John Doe"`
-	Schema JSONString `json:"schema" defaults:"{\"$schema\":\"https://json-schema.org/draft/2020-12/schema\",\"title\":\"Packages\",\"description\":\"A list of packages.\",\"type\":\"array\",\"items\":{\"type\":\"object\",\"properties\":{\"name\":{\"type\":\"string\"},\"version\":{\"type\":\"string\"},\"lock_file\":{\"type\":\"string\"},\"manager\":{\"type\":\"string\"},},\"required\":[\"name\",\"version\"],},\"minItems\":0,\"uniqueItems\":true}"`
+	Name   string     `json:"name" default:"John Doe"`
+	Schema JSONString `json:"schema,string" default:"\"{\\\"$schema\\\":\\\"https://json-schema.org/draft/2020-12/schema\\\",\\\"title\\\":\\\"Packages\\\",\\\"description\\\":\\\"A list of packages.\\\",\\\"type\\\":\\\"array\\\",\\\"items\\\":{\\\"type\\\":\\\"object\\\",\\\"properties\\\":{\\\"name\\\":{\\\"type\\\":\\\"string\\\"},\\\"version\\\":{\\\"type\\\":\\\"string\\\"},\\\"lock_file\\\":{\\\"type\\\":\\\"string\\\"},\\\"manager\\\":{\\\"type\\\":\\\"string\\\"},},\\\"required\\\":[\\\"name\\\",\\\"version\\\"],},\\\"minItems\\\":0,\\\"uniqueItems\\\":true}\""`
```

Comment:
Non-blocking

Is it possible for us to use the the JSON/JSONString type here combined with `fmt.Sprintf()` to set the tags? It would be easier to read.

Reply:
We can not. The golang spec says the struct [Tag](https://go.dev/ref/spec#Tag) must be a [sting literal](https://go.dev/ref/spec#string_lit). Basic string concatenating doesn't work either